app/javascript/components/widgets/numbers/falling-blocks.jsx

```diff
@@ -143,6 +160,33 @@ class Scene extends React.Component {
     'count',
   ]);
 
+  getImageFromCache(imagePath) {
+    let image = this.imageCache[imagePath];
+
+    if (image) {
+      return image;
+    }
+    image = new Image();
+    this.imageCache[imagePath] = image;
+    image.src = imagePath;
+
+    return image;
+  }
+
+  invisify = body => {
+    if (body.render.visible) {
+      const b = body;
+      b.render.visible = false;
+    }
+  };
+
+  invisify = body => {
+    if (body.render.visible) {
+      const b = body;
+      b.render.visible = false;
+    }
+  };
```

Comment:
This doesn't relate to the class so it can be extracted. 

Also can likely be simplified with a ramda method.

Also duplicated